frontend/src/api/DeviceApi.js

```diff
@@ -47,7 +47,8 @@ export function getSenders(callback) {
             sender.device.displayName,
             getStatus(sender.lastCommunication),
             channels,
-            ["Additional Device details go here"]
+            ["Additional Device details go here"],
+            "encoder"
```

Comment:
I think that "extras" should always be the last argument in the list, since it isn't actually being used yet and isn't a "real" value, whereas the "encoder" value is actually useful.

Reply:
none in particular, i just wasn't super sure where else to put it... immediately before extras?

Reply:
yeah, I think it would be better to have it before extras. Since, if we remove extras, which has been mentioned as an option, it'll be easier if it's at the end

Reply:
+1 Agree with Jun, after that change is made I can give my approval aswell

Reply:
moved it up one! b2e1224